modules/helm-installing-helm.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/getting-started-with-helm-on-openshift.adoc
+
+[id='installing-helm_{context}']
+= Installing Helm
+
+.Prerequisites:
+* You have installed Go version 1.13.
+
+. To install Helm, download the binary and create the Helm binary in the `/bin` directory with the `make` command:
```

Comment:
This is confusing to say to download the binary and then create the binary. I assume that if you are creating the binary, then it's not a binary that you're downloading, it's source code?

Reply:
You're right. Reworded it.